internal/service/connect/routing_profile.go

```diff
@@ -78,6 +78,12 @@ func resourceRoutingProfile() *schema.Resource {
 							Required:     true,
 							ValidateFunc: validation.IntBetween(1, 10),
 						},
+						"behaviour": {
```

Comment:
To match the AWS API, this should be a list nested block (`MaxItems: 1`) with a single string attribute inside. 

```hcl
  cross_channel_behavior {
    behavior_type = ""
  }
```